LEGACY/Assets/Plugins.meta

```diff
@@ -0,0 +1,6 @@
+fileFormatVersion: 2
```

Comment:
I know you want to emphasize the legaciness of this, but I would remove the all caps part of this path.

Reply:
Okay, noted!